crypto/pkcs7/pkcs7.c

```diff
@@ -1159,8 +1159,10 @@ static int pkcs7_decrypt_rinfo(unsigned char **ek_out, PKCS7_RECIP_INFO *ri,
   // we still need to set |ek| to NULL to signal decryption failure to callers
   // so they can use random bytes as content encryption key for MMA defense.
   if (!ok) {
+#if !defined(BORINGSSL_UNSAFE_FUZZER_MODE)
```

Comment:
suggest adding a comment here explaining why we don't want to null out `EK` here when fuzzing

Reply:
I removed the macro here.